.github/workflows/e2e-local.yml

```
@@ -54,10 +64,12 @@ on:
         default: false
 
 # Single global concurrency group — only one e2e run at a time.
-# The persistent instance can only serve one job. Newer pushes cancel older runs.
+# The persistent instance can only serve one job. Queued runs wait so isolated
+# database and VM cleanup cannot be interrupted by a newer push.
 concurrency:
   group: e2e-runner
-  cancel-in-progress: true
+  queue: max
```

actionlint (1.7.12)

[error] unexpected key "queue" for "concurrency" section. expected one of "cancel-in-progress", "group"

(syntax-check)